12_MeshLoaders/main.cpp

```diff
@@ -446,6 +438,24 @@ class MeshLoadersApp final : public MonoWindowApplication, public BuiltinResourc
 			return true;
 		}
 
+		void writeGeometry()
+		{
+			// make save path
+			static const auto prefix = std::filesystem::absolute("saved/");
+
+			if (!std::filesystem::exists(prefix))
+				m_system->createDirectory(prefix);
```

Comment:
path is wrong, see the 4 CWD members of the apocalypse in the basic framework app